### ISSUE TITLE
Resetting instance

### ISSUE THREAD

**Issue by mkucharz** [Syncano/syncano-node#64]
It would be nice to have a chance to reset instance instead of deleting/creating one. By resetting i mean deleting everything inside and cleaning up all the settings.

**Comment by 23doors** [comment]
@mkucharz What's the difference though? You could do DELETE and POST with same name (as you can specify new instance name through API) - even in 1 batch which would have the same effect.

**Comment by mkucharz** [comment]
@23doors actually I was hoping to speed up that process, I need mainly for test purposes. Creating instance takes some time - do you think we can speed it up?

I have already quite a lot E2E tests I want to run in parallel. To to that I have to create instance for each test to not interrupt other tests.

**Comment by 23doors** [comment]
I'll look into that but it may be difficult to speed up.
